javascript/new-src/src/factories/ola.js

```diff
@@ -22,6 +22,19 @@
 ola.factory('$ola', ['$http', '$window', 'OLA',
   function($http, $window, OLA) {
     'use strict';
+    var highestChannelNumberUsed = 0;
+
+    var updatehighestChannelNumberUsed = function(dmx) {
```

Comment:
Can you just check if OLAd has two universes patched this value is correctly kept local to each one?

Reply:
same here: changing the universe resets highestChannelNumberUsed